lib/model.js

```diff
@@ -2243,6 +2243,7 @@ class Model {
    * @param  {Boolean}      [options.benchmark=false] Pass query execution time in milliseconds as second argument to logging function (options.logging).
    * @param  {Boolean}      [options.returning=false] Append RETURNING * to get back auto generated values (Postgres only)
    * @param  {String}       [options.searchPath=DEFAULT] An optional parameter to specify the schema search_path (Postgres only)
+   * @param  {Array}        [options.preserveValuesOnNull]   Fields to preserve current values if row key already exists (on duplicate key update) and the value in the record is NULL? (only supported by mysql). By default, no fields are preserved.
```

Comment:
Add this option after `updateOnDupicate`, its description should day

```
Fields whose old data will be preserved when on duplicate key case try to replace it will null data. (Only available for MySQL)
```